portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,30 @@
+// Copyright 2019 Google LLC
```

Comment:
Do you still need this Comment class?

Reply:
The way I create the comments list in the Dataservlet it turns out I am not using the Comment class because I am making a list of Strings instead of a list of Comments.. so in this situation no. So I can either change the list to be of Comments, or delete this class and treat comments as Strings. Do you think it's better to have a Comment type?

Reply:
Generally, it's better to have classes represent objects; but in your case you do not expect to expand the Comment class anymore (especially as this project is temporary :) 
So I'd vote for removing the Comment class for now.